features/machine/machine.feature

```diff
@@ -114,3 +114,32 @@ Feature: Machine features testing
     Given I have an IPI deployment
     And evaluation of `BushSlicer::Machine.list(user: admin, project: project('openshift-machine-api'))` is stored in the :machines clipboard
     Then the expression should be true> cb.machines.select{|m|m.instance_state == m.annotation_instance_state}.count == cb.machines.count
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-27609
+  @admin
+  @destructive
+  Scenario: Scaling a machineset with providerSpec.publicIp set to true
+    Given I have an IPI deployment
+    And I clone a machineset named "machineset-clone-publiciptrue"
+
+    Then I run the :get admin command with:
+     | resource          | machineset                       |
+     | resource_name     | machineset-clone-publiciptrue    |
+     | namespace         | openshift-machine-api            |
+     | o                 | yaml                             |
```

Comment:
Unfortunately, the extra meaningless white-spaces is still there and within below lines.

Reply:
I have left that space for readability , Please let me know the line numbers which you feel unnecessary spacing is there will remove those

Reply:
All the lines with tables. 
I'm not sure how do you check that, but I'm checking in the web, maybe there are white-space/tab-space differences.

Reply:
I used - https://github.com/openshift/verification-tests/blob/master/doc/vim.adoc , but dont see any alignment issues

Reply:
A result like this will look better.

Before
 ```
| resource_name     | machineset-clone-publiciptrue    |
```

After 
```
| resource_name | machineset-clone-publiciptrue |
```

Reply:
@miyadav it's the extra spaces between the names and `|`

Reply:
Thanks made the change for spacing .. done